pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -194,9 +196,15 @@ func NewAttachDetachController(
 		adc.intreeToCSITranslator)
 
 	podInformer.Informer().AddEventHandler(kcache.ResourceEventHandlerFuncs{
-		AddFunc:    adc.podAdd,
-		UpdateFunc: adc.podUpdate,
-		DeleteFunc: adc.podDelete,
+		AddFunc: func(obj interface{}) {
+			adc.podAdd(ctx, obj)
```

Comment:
The context is always going to be the same. Better look up the logger once and pass that.

Same below.